lib/reporters/base.js

```diff
@@ -169,7 +169,14 @@ exports.list = function(failures) {
     // msg
     var msg;
     var err = test.err;
-    var message = err.message || '';
+    var message;
+    if (err.message) {
+      message = err.message;
+    } else if (typeof err.inspect === 'function') {
+      message = err.inspect();
```

Comment:
The return value of an object's inspect function doesn't have to be a string. Would be worth calling toString to ensure that message is always a string.

From the docs: https://nodejs.org/api/util.html#util_util_inspect_object_options

> You may also return another Object entirely, and the returned String will be formatted according to the returned Object. This is similar to how JSON.stringify() works:

```
var obj = { foo: 'this will not show up in the inspect() output' };
obj.inspect = function(depth) {
  return { bar: 'baz' };
};

util.inspect(obj);
  // "{ bar: 'baz' }"
```